src/isosceles-triangle.js

```diff
@@ -0,0 +1,20 @@
+function drawIsoscelesTriangle(h) {
+    let shape = "";
+    for(let i = 1; i <= h; i++) {
+        // print space
+        for(let j = 1; j<= h-i; j++) {
+            shape += " ";
+        }
+        // print star "*"
+        for(let k = 1; k <= 2*i-1; k++) {
+            shape += "*";
+        }
+        // print space
+        for(let j = 1; j<= h-i; j++) {
```

Comment:
unnecessary for loop => remove this loop

Reply:
okay